config/webpack.config.js

```diff
@@ -51,7 +51,7 @@ const commonConfig = ({ dev }) => {
           },
         }
       : {}),
-    devtool: false,
+    devtool: process.env.BRANCH === 'prod-beta' ? 'hidden-source-map' : false,
```

Comment:
@karelhala do you remember if we have this env variable in the containerized build?